packages/svg/src/icons/ApplicationIcon.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC } from "react";
+import { IconProps } from "./types";
+
+const ApplicationIcon: FC<IconProps> = ({
```

Comment:
we should separate solid, outline and  color icons so we dont  have  some icons doing  one thing and others different